src/test/java/core/basesyntax/service/impl/DataConverterServiceImplTest.java

```diff
@@ -0,0 +1,133 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverterService;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataConverterServiceImplTest {
+    private static final List<String> operationsValid1 =
+            readFileContents("src/test/resources/operations_valid_1.csv");
+    private static final List<String> operationsValid2 =
+            readFileContents("src/test/resources/operations_valid_2.csv");
+    private static final List<String> operationsEmpty =
+            readFileContents("src/test/resources/operations_empty.csv");
+    private static final List<String> operationsMixedFields =
+            readFileContents("src/test/resources/operations_mixed_fields.csv");
+    private static final List<String> operationsRandomFields =
+            readFileContents("src/test/resources/operations_random_text.csv");
+    private static final List<String> operationsWrongRegex =
+            readFileContents("src/test/resources/operations_random_text.csv");
+    private static DataConverterService dataConverter = new DataConverterServiceImpl();
+
+    private FruitTransaction transaction1;
+    private FruitTransaction transaction2;
+    private FruitTransaction transaction3;
+    private FruitTransaction transaction4;
+    private List<FruitTransaction> transactions;
+
+    @BeforeEach
```

Comment:
I think you should make `FruitTransaction` objects local - they're only used in few tests, so  the usage of `@BeforeEach` may be misleading.